app/src/main/java/com/criteo/testapp/StandaloneActivity.java

```diff
@@ -57,19 +54,24 @@ protected void onCreate(Bundle savedInstanceState) {
     setContentView(R.layout.activity_stand_alone);
     MockedIntegrationRegistry.force(Integration.STANDALONE);
 
-    adLayout = findViewById(R.id.AdLayout);
     btnShowInterstitial = findViewById(R.id.buttonStandAloneInterstitial);
     btnShowInterstitialIbv = findViewById(R.id.buttonStandAloneInterstitialIbv);
+    criteoBannerView = findViewById(R.id.criteoBannerView);
+    criteoBannerView.setCriteoBannerAdListener(new TestAppBannerAdListener(
+        TAG, "Standalone"));
 
     context = getApplicationContext();
 
     nativeLoader = new CriteoNativeLoader(
         NATIVE,
-        new TestAppNativeAdListener(TAG, NATIVE.getAdUnitId(), adLayout),
+        new TestAppNativeAdListener(TAG, NATIVE.getAdUnitId()),
         new TestAppNativeRenderer()
     );
 
-    findViewById(R.id.buttonStandAloneBanner).setOnClickListener(v -> loadBanner());
+    findViewById(R.id.buttonStandAloneBanner).setOnClickListener(v -> {
+      Log.d(TAG, "Banner Requested");
```

Comment:
Nit: Same comment than in `InHouseActivity`. IMHO Having really light lambda keep the call site more readability.